libs/remix-astwalker/tests/newTests.ts

```diff
@@ -130,12 +130,12 @@ tape("New ASTWalker", (t: tape.Test) => {
 	 However, for non-typescript uses, we add this test which casts
 	 to an AST to check that there is a run-time check in walkFull.
       */
-      astWalker.walkFull(<AstNode>legacyNode, (node: AstNode) => {
+      astWalker.walkFull(<any>legacyNode, (node: AstNode) => {
```

Comment:
is that fine to have this `<any>` here?

Reply:
Because a there is type definition mismatch for legacyAST node as we don't support that now